Apply proofread changes (#107)
infinum/wordpress-handbook: Automated testing in WordPress/Core tests.md

Here we've specified that code coverage should be created. For that to be done, you need to have Xdebug installed on your system. Keep in mind that Xdebug may significantly reduce the speed of your tests.

> In the root directory—next to `wp-tests-config.php`, the `tests/` folder, and the `phpunit.xml.dist` file—run:

Reviewer: ```suggestion
In the root directory, where `wp-tests-config.php` file, the `tests/` folder, and the `phpunit.xml.dist` file are located, run:
```
